views/tracks.hbs

```diff
@@ -0,0 +1,12 @@
+<a href="http://localhost:3000">Back to search</a>
```

Comment:
this is not good. The main reason why is because it is specific to your machine and will break on a deployed machine. 
instead use relative urls! i.e. href="/" which would be the homepage, no matter where stuff is hosted. I.e. if your host is mydomain.com -> "/" would still work -> whereas localhost:3000 would not.